dlt/helpers/streamlit_app/blocks/explorer.py

```diff
@@ -0,0 +1,43 @@
+import dlt
+import pygwalker
+import streamlit as st
+from pygwalker.api.streamlit import StreamlitRenderer
+
+st.set_page_config(layout="wide")
+
+
+
+
+
+@st.cache_resource
+def pygwalker_renderer(pipeline_name: str, table_name: str) -> StreamlitRenderer:
+    pipeline = dlt.attach(pipeline_name)
+    dataset = pipeline.dataset()
+    df = dataset[table_name].df()
```

Comment:
this will load the full table into memory, is this desired? on a large table this will take a long time and at some point kill the host. is there some other way we could do this?